packages/ubus-lime-fbw/files/www/cgi-bin/hostname

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+echo "Status: 200"
+echo "Content-type: text/plain;"
+echo "Cache-control: max-age=0, no-cache;"
+echo "Access-Control-Allow-Origin: *"
+echo "Access-Control-Allow-Methods: GET;"
+echo "Access-Control-Allow-Headers: X-PINGOTHER, Content-Type;"
+echo
+echo $HOSTNAME
```

Comment:
Just a thought, in the future if we start adding more of this we may refactor into something more maintainable than than bash/cgi.

Reply:
Yeah, we can do a lot more about it. For now I would leave it at that and see if I could find a more general solution.